turbo/generators/templates/env/src/env.d.ts

```
@@ -0,0 +1,9 @@
+declare global {
+	namespace NodeJS {
+		interface ProcessEnv extends ExtendEnvironment {}
+	}
+}
+
+interface ExtendEnvironment {}
```

Biome (2.4.4)

[error] An empty interface is equivalent to {}.

(lint/suspicious/noEmptyInterface)